tests/mocks/chapters.ts

```diff
@@ -1,9 +1,10 @@
 export const chapter = {
   id: 1,
   chapterNumber: 1,
+  surahCaligraphy: '001',
   bismillahPre: false,
   revelationOrder: 5,
-  revelationPlace: 'makkah',
+  revelationPlace: 'Mecca',
```

Comment:
mocked data should resemble remote API response. it's `makkah` and `madinah` from the API